### ISSUE TITLE
AnimationTree - Object was deleted while awaiting a callback

### ISSUE THREAD

**Issue by ChaqSilent** [godotengine/godot#41512]
The error "Object was deleted while awaiting a callback" appears when I try to edit a blend tree graph node inside an AnimationTree:

I use Godot 3.2.2

I have an AnimationTree node in my scene. The root of the AnimationTree node is an AnimationNodeBlendTree. In the graph editor, there are OneShot nodes, Transition nodes, Blend2 and Blend3 nodes, a StateMachine, TimeScales - and two BlendTree nodes.

Each of those two BlendTree nodes inside the AnimationTree (which has an AnimationNodeBlendTree root) freezes the engine when I want to edit them. Only once in a while I am able to open those two BlendTree nodes when I press the edit button, but the rest of the time the engine hangs.

![IMG-20200825-WA0001](https://user-images.githubusercontent.com/55622264/91203654-875a9480-e703-11ea-8599-d3b5affdcdaf.jpg)

**Comment by Calinou** [comment]
Duplicate of #35653 (the underlying issue is the same).
